python/tank/util/user_settings.py

```diff
@@ -269,7 +270,15 @@ def _load_config(self, path):
 
         :returns: A ConfigParser instance with the contents from the configuration file.
         """
-        config = configparser.SafeConfigParser()
+        # In Python 3.2, SafeConfigParser has been renamed to ConfigParser and using the
+        # old class name generates a warning.
+        if six.PY2:
+            config = configparser.SafeConfigParser()
+        else:
+            # Technically this only appeared in Python 3.2, but we don't support
+            # less than 3.7, so we don't have to be very precise about which
+            # version of Python 3 we are running.
+            config = configparser.ConfigParser()
```

Comment:
Again, is this something to consider adding to sgsix?

Reply:
This is the only time we're using this. Let's not put too much stuff in there. I don't even think we use this class anywhere else in Toolkit applications or frameworks!